packages/contracts-bedrock/deploy-config/devnetL1-template.json

```diff
@@ -50,7 +50,7 @@
   "requiredProtocolVersion": "0x0000000000000000000000000000000000000000000000000000000000000000",
   "recommendedProtocolVersion": "0x0000000000000000000000000000000000000000000000000000000000000000",
   "faultGameAbsolutePrestate": "0x03c7ae758795765c6664a5d39bf63841c71ff191e9189522bad8ebff5d4eca98",
-  "faultGameMaxDepth": 46,
+  "faultGameMaxDepth": 50,
```

Comment:
@clabby The newly added e2e tests sometimes flakes due to insufficient execution depth. I'm bumping this up just a bit to avoid those flakes.

Reply:
ack - no worries.